### ISSUE TITLE
Editors does not break a line of Thai text properly.

### ISSUE THREAD

**Issue by veer66** [ONLYOFFICE/DesktopEditors#371]
Editors does not break a line of Thai text properly. Line breaking must not breaks word (in Thai).  

**What is the current behavior?**

ดูการบ้านดูการบ้านดูการบ้าน has 6 words that are ดู|การบ้าน|ดู|การบ้าน|ดู|การบ้าน. Currently Editors break a line at ดูการบ้านดูการบ้านดูกา|รบ้าน.

![ภาพ](https://user-images.githubusercontent.com/370688/80122621-fc805680-85b7-11ea-8591-6c436369d8c7.png)

**What is the expected behavior?**

The line must be split as follow ดูการบ้านดูการบ้านดู|การบ้าน.

![ภาพ](https://user-images.githubusercontent.com/370688/80122862-5123d180-85b8-11ea-9739-79928bbf9a86.png)

ONLYOFFICE Desktop Editors version 5.5.1.78
Licensed under GNU AGPL v3

Windows 10

[thai-bug-2.docx](https://github.com/ONLYOFFICE/DesktopEditors/files/4523769/thai-bug-2.docx)

**Comment by KirillovIlya** [comment]
Hello. Can you make another screenshot in MSWord with top ruler enabled?

**Comment by veer66** [comment]
@KirillovIlya No, I have no idea what top ruler is and I don't know how to enable it.

How about LibreOffice?

![ภาพ](https://user-images.githubusercontent.com/370688/80274255-47f34b80-8703-11ea-8b88-0bebac96b15c.png)

I suppose you can reproduce by copy & paste this string "ดูการบ้านดูการบ้านดูการบ้าน" and set font to Tahoma 48.

**Comment by KirillovIlya** [comment]
It seems we need Thai support. In this language, words are not divided by a special character, such as a space, but according to their own rules.

**Comment by Rita-Bubnova** [comment]
@veer66, thank you for report.
I create new issue 45287 in our private issue tracker.
